src/DynamoCore/DynamoCore.csproj

```diff
@@ -74,13 +74,17 @@ limitations under the License.
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\..\extern\CSharpAnalytics\CSharpAnalytics.WinForms.dll</HintPath>
     </Reference>
-    <Reference Include="Greg, Version=1.0.0.0, Culture=neutral, processorArchitecture=MSIL">
+    <Reference Include="Greg, Version=1.0.5892.25330, Culture=neutral, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\..\extern\greg\Greg.dll</HintPath>
```

Comment:
@aparajit-pratap Yes. We're building Dynamo with Travis now. I didn't have a Greg nuget at the time on the public feed, so Travis's package restore wouldn't have been able to find it. I've since posted a public nuget for Greg, so we could do this.